pkg/core/interop_neo.go

```diff
@@ -49,7 +49,10 @@ func storageFind(ic *interop.Context) error {
 
 	filteredMap := stackitem.NewMap()
 	for k, v := range siMap {
-		filteredMap.Add(stackitem.NewByteArray(append(prefix, []byte(k)...)), stackitem.NewByteArray(v))
+		key := append(prefix, []byte(k)...)
+		keycopy := make([]byte, len(key))
```

Comment:
We can simply copy `prefix` and `k` to `keycopy`. This will save us 2 allocations (if we also replace `[]byte(k)` with `k`).
Some test is also desired.